A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_AOC_SIMBRIEF.js

```diff
@@ -2,7 +2,7 @@ class CDU_OPTIONS_SIMBRIEF {
     static ShowPage(mcdu) {
         mcdu.clearDisplay();
 
-        const simbriefUsername = NXDataStore.get("CONFIG_SIMBRIEF_USERNAME", "");
+        const simbriefUsername = NXDataStore.get("CONFIG_SIMBRIEF_USERNAME", "").replace('_', ' ');
 
         const simbriefUsernameString = simbriefUsername != "" ? simbriefUsername : "[ ]";
```

Comment:
```suggestion
        const simbriefUsername = NXDataStore.get("CONFIG_SIMBRIEF_USERNAME", "");

        const simbriefUsernameString = simbriefUsername ? simbriefUsername..replace('_', ' ') : "[ ]";
```

Reply:
or use `-` instead

Reply:
Is it not cleaner to store the correct username (ie. with underscores) in the DataStore, and deal with rewriting it when saving/displaying it? This way around, anywhere using  NXDataStore.get("CONFIG_SIMBRIEF_USERNAME") doesn't need to know what character we're substituting for underscores, we can keep that here instead.

Also, is '-' allowed in Simbrief usernames? I'd assumed it was legal which was why I didn't use it, but happy to switch it around if it's not.

Reply:
this doesn't modify the stored data, just the displayed one

Reply:
The modification happens further down when the user enters their username though